### ISSUE TITLE
[Screen Reader-Projects-My Stuff-Import]: Screen reader not providing any information about the error message on Import Dialog box.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2526]
**Pre-Requisite:** Turn on Narrator(Win + Ctrl + Enter)

**User Experience:**
Users who depends on Screen Reader will get confuse if any screen reader will not provide the information about the any error message showing on Import dialog box.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Projects control lying in the header section on the page and select it.
4. Navigate to the MY Stuff tab on the page opened and select Import option.
5. Navigate to various controls lying on the page opened.
6. Verify that screen reader is not providing any information about the Error message or not.

**Actual Result:-**
After uploading any invalid file screen reader is not providing any information about the Error message shown.

**Expected Result:-**
After uploading any invalid file screen reader should provide information about the Error message shown.

**Note:** This same issue exists on NVDA + FF
 
**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Please refer the attachment**
[MAS1.3.1_Projects_Import file.zip](https://github.com/Microsoft/pxt/files/1176111/MAS1.3.1_Projects_Import.file.zip)

![mas1 3 1_projects_import file](https://user-images.githubusercontent.com/30459830/28614408-5d74a2d6-7213-11e7-9cc2-f904895c509f.jpg)

**Comment by abchatra** [comment]
@v-amik. Does this `[Screen Reader-Projects-My Stuff-Import]` needs to be a label? This does look like part of the title.

**Comment by veler** [comment]
@v-amik Issue fixed and available to test

**Comment by v-amik** [comment]
As per process, bug should be assigned to HCL for regression

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug.
